frontend/src/features/projects/common/ProjectSummaryView.test.tsx

```diff
@@ -89,6 +88,35 @@ const getMockProject = (statusCode?: string): IProject => ({
     isActive: false,
     workflowCode: '',
   },
+  marketedOn: '2020-01-01',
```

Comment:
these financials ensure that the calculated gainBeforeSpl and netProceeds will not change when the form loads from calculations. This allows the test to proceed without having to specifically wait for those calculations to update the dom.